Leibit.Core/Data/sued/RSI.xml

```diff
@@ -0,0 +1,330 @@
+    <track name="2" calculateDelay="false">
+      <block name="56B1"/>
+    </track>
+    <track name="1" calculateDelay="false">
+      <block name="56BP"/>
+    </track>
+  </station>
+
+  <station name="Singen (Hohentwiel) Industriegebiet" short="RSII" refNr="55" scheduleFile="b55_____.abf">
+    <track name="2" calculateDelay="false">
+      <block name="55BN1"/>
+    </track>
+    <track name="1" calculateDelay="false">
+      <block name="55B2"/>
+    </track>
+  </station>
+  
+  <station name="Singen (Hohentwiel)" short="RSI" refNr="55" scheduleFile="g55_____.abf" localOrderFile="bf55____.abf">
+    <track name="1">
+      <block name="55G1"/>
+    </track>
+    <track name="51">
+      <block name="55G1"/>
+    </track>
+    <track name="2">
+      <block name="55G2"/>
+    </track>
+    <track name="3">
+      <block name="55G3"/>
+    </track>
+    <track name="4">
+      <block name="55G4"/>
+    </track>
+    <track name="5">
+      <block name="55G5"/>
+    </track>
+    <track name="6">
+      <block name="55G6"/>
+    </track>
+    <track name="8">
+      <block name="55G8"/>
+    </track>
+    <track name="9">
+      <block name="55G9"/>
+    </track>
+    <track name="10">
+      <block name="55G10"/>
+    </track>
+    <track name="11">
+      <block name="55G11"/>
+    </track>
+    <track name="12">
+      <block name="55G12"/>
+    </track>
+    <track name="13">
+      <block name="55G13"/>
+    </track>
+    <track name="14">
+      <block name="55G14"/>
+    </track>
+    <track name="15">
+      <block name="55G15"/>
+    </track>
+    <track name="116">
+      <block name="55G116"/>
+    </track>
+  </station>
+
+  <station name="Gottmadingen" short="RG" refNr="54" scheduleFile="g54_____.abf" localOrderFile="bf54____.abf">
+    <track name="1">
+      <block name="54G1"/>
+    </track>
+    <track name="2">
+      <block name="54G2"/>
+    </track>
+  </station>
+
+  <station name="Bietingen" short="RBGN" refNr="54" scheduleFile="b54_____.abf">
+    <track name="1" calculateDelay="false">
+      <block name="53BN"/>
+    </track>
+    <track name="2" calculateDelay="false">
+      <block name="54BP"/>
+    </track>
+  </station>
+
+  <station name="Thayngen" short="RTG" refNr="53" scheduleFile="g53_____.abf" localOrderFile="bf53____.abf">
+    <track name="1">
+      <block name="53G1"/>
+    </track>
+    <track name="2">
+      <block name="53G2"/>
+    </track>
+    <track name="3">
+      <block name="53G3"/>
+    </track>
+  </station>
+
+  <station name="Herblingen" short="RHRB" refNr="53" scheduleFile="herbli__.abf">
+    <track name="2" calculateDelay="false">
+      <block name="53BP"/>
+    </track>
+    <track name="1" calculateDelay="false">
+      <block name="52BN"/>
+    </track>
+  </station>
+
+  <station name="Schaffhausen-Fulachkanal" short="RSCFO" refNr="52" scheduleFile="g52_____.abf" localOrderFile="bf52____.abf">
```

Comment:
Die Zuordnungen hier beißen sich mit denen von Herblingen. Ich hatte ein ähnliches Problem in NGM mit dem Abzw. Zollberg und dem Hp Rieneck. Dort habe ich dem Abzw. den Vortritt gelassen, weil dort auch sinnvoll Verspätung berechnet werden kann. Den Hp habe ich trotzdem drin gelassen, allerdings nur mit einem "leeren" Gleis ohne Block-Zuordnung. Ich weiß noch nicht, ob diese Idee so gut ist, bzw. welche Nebenwirkungen sie hat... Ansonsten im Zweifel den Hp weglassen.

Reply:
Ich hab die Blöcke jetzt auch mal rausgenommen. ggf. hätte man die Blöcke auch nur für die Richtung, entgegen des Abzweigs nutzen können?

Reply:
Sehr gut. Ja, wäre wahrscheinlich gegangen, aber dann versteht es keiner mehr, wenn das nur in eine Richtung funktioniert...